app/views/puppetclasses/_class_parameters.html.erb

```diff
@@ -1,42 +1,26 @@
 <% lookup_keys = overridable_lookup_keys(puppetclass, obj) %>
-<% value_hash  = value_hash_cache(obj) if obj.class.model_name == "Host" %>
-<% can_edit    = authorized_via_my_scope("host_editing", "edit_params") %>
-<% lookup_keys.each_with_index do |key,i| %>
-  <% overridden = key.overridden?(obj) %>
-  <% disabled = can_edit && !overridden %>
-  <% lookup_value = obj.lookup_values.detect{|v| v.lookup_key_id == key.id} %>
-  <% error = lookup_value.try(:errors) %>
```

Comment:
All .try calls which hide the problem of knowing whether the object is there or not, I tried to fix it by calling `LookupValue.new` when the lookup_key has not been overridden yet